skyve-war/src/main/java/modules/admin/Configuration/Configuration.xml

```diff
@@ -68,6 +68,11 @@
 			<displayName>admin.configuration.selfRegistrationActivationExpiryHours.displayName</displayName>
 			<description>admin.configuration.selfRegistrationActivationExpiryHours.description</description>
 		</integer>
+		<integer name="passwordResetTokenExpiryMinutes">
+			<displayName>Password Reset Token Expiry (minutes)</displayName>
+			<description>Clear this setting to have tokens that never expire.</description>
+			<defaultValue>15</defaultValue>
+		</integer>
```

Comment:
This should have a `validator` with a min value of 1 so that we do not need to handle 0 or a negative expiry time in minutes.